src/worker/exporters/node_exporter.py

```diff
@@ -287,7 +290,9 @@ def init_config(job_id, port=None):
         'job_id': job_id,
         'fieldFiles': {},
         'counter': {},
-        'event_timestamp': datetime.datetime.now().strftime("%b %d %H:%M:%S")
+        'sample_timestamp': {},
+        'labels': {},
+        'columns': {}
```

Comment:
Why do we need `columns` here?

Reply:
Ah this was left over from the custom metric addition. Let me remove as well